storage/src/adb/any/sync/client.rs

```diff
@@ -252,13 +252,17 @@ where
                 );
 
                 // Get proof and operations from resolver
-                let GetProofResult {
+                let GetOperationsResult {
                     proof,
                     operations,
                     success_tx,
                 } = {
                     let _timer = metrics.fetch_duration.timer();
-                    config.resolver.get_proof(log_size, batch_size).await?
+                    let target_size = config.upper_bound_ops + 1;
```

Comment:
Curious where the `+1` comes from here? I would've expected us to go right to upper_bound (unless that is meant to be non-inclusive)?